README.md

```diff
@@ -91,4 +92,6 @@ watch                       Starts the example app and performs the hot reload i
 
 ### Publish to pub.dev
 
-To make the plugins accessible, we want to publish them to pub.dev. [publish_all_packages.yml](.github/workflows/publish_all_packages.yml) contains a job that deploys all four plugins to pub.dev. This job is named `Publish all packages to pub.dev` and is triggered manually. If you want to check first if it's safe to publish the plugins, you can run `dart pub publish --dry-run` in the root of each plugin.  
+To make the plugins accessible, we plan to publish them to pub.dev. The first step in this process is to create Git tags, which is required for publishing. This can be done by triggering the [create-git-tags-for-publishing.yml](.github/workflows/create-git-tags-for-publishing.yml) workflow. This action generates Git tags for all packages in the repository.
```

Comment:
Maybe change the "we plan to publish...", since we accomplished this now and it actually is already published 😁